packages/expo/src/provider/singleton/createClerkInstance.ts

```diff
@@ -0,0 +1,81 @@
+import type { FapiRequestInit, FapiResponse } from '@clerk/clerk-js/dist/types/core/fapiClient';
+import type { Clerk } from '@clerk/clerk-js/headless';
+import type { BrowserClerk, HeadlessBrowserClerk } from '@clerk/clerk-react';
+
+import { MemoryTokenCache } from '../../cache/MemoryTokenCache';
+import type { TokenCache } from '../../cache/types';
+import { errorThrower } from '../../errorThrower';
+
+type BuildClerkOptions = {
+  publishableKey?: string;
+  tokenCache?: TokenCache;
+};
+
+const KEY = '__clerk_client_jwt';
+
+/**
+ * @deprecated Use `getClerkInstance` instead. `Clerk` will be removed in the next major version.
+ */
+export let clerk: HeadlessBrowserClerk | BrowserClerk;
+let __internal_clerk: HeadlessBrowserClerk | BrowserClerk | undefined;
+
+/**
+ * Access or create a Clerk instance outside of React.
+ * @example
+ * import { ClerkProvider, getClerkInstance } from "@clerk/expo"
+ *
+ * const clerkInstance = getClerkInstance({ publishableKey: 'xxxx' })
+ *
+ * // Always pass the `publishableKey` to `ClerkProvider`
+ * <ClerkProvider publishableKey={'xxxx'}>
+ *     ...
+ * </ClerkProvider>
+ *
+ * // Somewhere in your code, outside of React you can do
+ * const token = await clerkInstance.session?.getToken();
+ * fetch('http://example.com/', {headers: {Authorization: token })
+ * @throws MissingPublishableKeyError publishableKey is missing and Clerk has not been initialized yet
+ * @returns HeadlessBrowserClerk
+ */
+export function createClerkInstance(ClerkClass: typeof Clerk) {
+  return (options?: BuildClerkOptions): HeadlessBrowserClerk | BrowserClerk => {
+    const { publishableKey = process.env.CLERK_PUBLISHABLE_KEY || '', tokenCache = MemoryTokenCache } = options || {};
```

Comment:
Does this need the `EXPO_PUBLIC` prefix?

Reply:
Nice catch, probably didn't include it when I was doing rebase 😅